.gitignore

```diff
@@ -14,4 +14,6 @@
 .cxx
 local.properties
 /.idea/
-*.log
\ No newline at end of file
+*.log
+/default_config/
+**/config_settings.yaml
```

Comment:
Please remove `/default/config/` from gitignore. 
And update the `**/config_settings.yaml` to `/config_settings.yaml`